packages/lpa-questionnaire-web-app/src/routes/authentication.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
```

Comment:
@LouiseMcMahon I know that when we spoked about the endpoints you were thinking to group the pages under the '/authentication/' path but since then the ticket has been updated and now it expects the following endpoints.  

'/your-email'
'/confirm-email'

I have kept your initial idea as I think is better to group the pages together, since they will never be used independently. Also the endpoint is more descriptive. 

We should chat with Pippa tomorrow and find out if this is ok.

Reply:
Agreed, however Pippa on holiday.

Reply:
Anthony and the content team has approved this change.